xml/docu_styleguide.language.xml

```diff
@@ -27,9 +27,9 @@
  <sect2 xml:id="sec-abbreviation">
   <title>Abbreviations</title>
   <para>
-   Where possible, avoid using abbreviations. Especially avoid unusual
-   abbreviations. You may create plurals of acronyms. In all other cases,
-   avoid creating plurals of abbreviations.
+   Where possible, avoid using abbreviations, especially unusual
```

Comment:
```suggestion
   Avoid using abbreviations, especially unusual
```

Reply:
Done! Are the other fine with the PR as it is now?